packages/api/src/lcd.ts

```diff
@@ -24,6 +53,33 @@ class API {
     this.ownership = new Ownership(endpoint)
     this.account = new Account(endpoint)
   }
+
+  async createTransaction(
+    msgs: IMsg<any>[],
+    account: IAccount,
+    opts: { fee?: IFee, chain_id?: string, account_number?: number, sequence?: number, memo?: string } = {}) {
+    return new Transaction({
+      account_number: opts.account_number
+        ? opts.account_number.toString()
+        : account.account_number.toString(),
+      chain_id: opts.chain_id || (await this.getRequest(`/node_info`)).node_info.network,
+      fee: opts.fee || {
+        amount: [{ denom: 'atto', amount: '300000' }],
+        gas: '300000'
```

Comment:
is there an lcd api to simulate the tx and get the fee?

Reply:
Yes there is, I dont want to complexify it yet

Reply:
https://github.com/mesg-foundation/js-sdk/issues/185